atomic_reactor/plugins/build_orchestrate_build.py

```diff
@@ -17,6 +17,7 @@
 from atomic_reactor.build import BuildResult
 from atomic_reactor.plugin import BuildStepPlugin
 from atomic_reactor.plugins.pre_reactor_config import get_config
+from atomic_reactor.plugins.pre_add_filesystem import AddFilesystemPlugin
```

Comment:
Let's avoid this import by moving the value of `AddFilesystemPlugin.key` to constants.py.
Both this plugin and `add_filesystem` should read from the constants file. 
See how `PLUGIN_PULP_SYNC_KEY` is done.

The `add_filesystem` plugin requires koji library. 
We try to isolate such plugin in its own subpackage.
Not all installation of atomic-reactor will include `add_filesystem` plugin, for example.